src/Catty/ViewController/Upload/UploadViewController.swift

```diff
@@ -232,6 +257,43 @@ class UploadViewController: UIViewController {
         navigationItem.rightBarButtonItem = UIBarButtonItem(customView: barButtonSpinner)
     }
 
+    func createLabel(text: String, textColor: UIColor, font: UIFont) {
+        let label = UILabel()
+        label.text = text
+        label.textColor = textColor
+        label.font = font
+        self.view.addSubview(label)
+
+        if textColor == UIColor.globalTint {
```

Comment:
Please remove condition based on text color (changes to the color will affect the internal logic which is very error prone). Please make another function `createValue` (or similar).